test/e2e_node/util.go

```diff
@@ -17,11 +17,50 @@ limitations under the License.
 package e2e_node
 
 import (
+	"encoding/json"
 	"flag"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/golang/glog"
+
+	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats"
 )
 
 var kubeletAddress = flag.String("kubelet-address", "http://127.0.0.1:10255", "Host and port of the kubelet")
 
 var buildServices = flag.Bool("build-services", true, "If true, build local executables")
 var startServices = flag.Bool("start-services", true, "If true, start local node services")
 var stopServices = flag.Bool("stop-services", true, "If true, stop local node services after running tests")
+
+func getNodeSummary() (*stats.Summary, error) {
+	req, err := http.NewRequest("GET", "http://localhost:10255/stats/summary", nil)
```

Comment:
It would be better to also consolidate https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/kubelet_test.go#L122-L150